it/v3/endpoints/SubmitSelfEmploymentBsasControllerISpec.scala

```diff
@@ -126,7 +126,7 @@ class SubmitSelfEmploymentBsasControllerISpec extends IntegrationBaseSpec {
 
         val input = Seq(
           ("AA1234A", BAD_REQUEST, NinoFormatError, requestBody),
-         // ("AA123456A", BAD_REQUEST, RuleBothExpensesError, mtdRequestWithBothExpenses),
+          // ("AA123456A", BAD_REQUEST, RuleBothExpensesError, mtdRequestWithBothExpenses),
```

Comment:
This part needs to be rewritten so the validation error test covers the `RuleBothExpensesError`